integrations/opensearch/src/haystack_integrations/document_stores/opensearch/document_store.py

```diff
@@ -101,6 +127,7 @@ def to_dict(self) -> Dict[str, Any]:
             self,
             hosts=self._hosts,
             index=self._index,
+            max_chunk_bytes=self._max_chunk_bytes,
             **self._kwargs,
         )
```

Comment:
As the now explicitly added kwargs `embedding_dim`, `method`, `mappings` and `settings` are not part of `self._kwargs` anymore, we have to treat them separately here.
Can we please add them here too and add a proper test that ensures any init args are serialized correctly?

Reply:
good point! I added them now